tests/dubstep_dj_test.js

```diff
@@ -0,0 +1,12 @@
+import test from 'ava';
+import songDecoder from '../src/dubstep_dj.js';
+
+test('Should be equal', t => {
```

Comment:
Test name should be explanatory about what is tested and what is expected behaviour